src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1154,6 +1158,13 @@ MulticopterPositionControl::task_main()
 
 		poll_subscriptions();
 
+		/* get current rotation matrix and euler angles from control state quaternions */
+		math::Quaternion q_att(_ctrl_state.q[0], _ctrl_state.q[1], _ctrl_state.q[2], _ctrl_state.q[3]);
```

Comment:
I reverted this change below. I will resend separately with a justification if it is needed.